js/login.js

```diff
@@ -58,8 +58,6 @@ function ValidationPassword() {
 
 function ValidationFormulaire()
 {
-    ValidationEmail();
```

Comment:
La validation se fait déjà dans le return. Par contre, il ne feras pas la validation du mot de passe avant que ValidationEmail() soit true. Ce qui a du sens à mon avis, mais si tu souhaites faire les deux validations en même temps, on peut garder le code ainsi

Reply:
Non il le fait pas parce qu'il fait juste le premier AND quand il return false. Il faut valider les deux pour afficher les 2 erreurs.